tests/src/autoload/functions.php

```diff
@@ -188,6 +188,39 @@ function loadDataset()
                 'entities_id' => '_test_root_entity',
             ],
         ], 'User' => [
+            [
+                'id'            => 3, // Force id to not break existing tests
```

Comment:
I think it would be better to properly refactor the tests rather than force the old harcoded ids here.

It might lead to some issues when additional users are created (like with e2e tests)

Reply:
I agree. I tried to find a quick path to fix the tests, but they are anyway still failing.

IMHO, this requires some hours of refactoring. I am not sure we have time to do it right now.